modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -203,14 +203,13 @@ private static Map<String, Object> _getVariables(
 
 		Map<String, Object> objectEntry =
 			(Map<String, Object>)payloadJSONObject.get("objectEntry");
-		String userId = payloadJSONObject.getString("userId");
 
 		Map<String, Object> allowedVariables =
 			HashMapBuilder.<String, Object>put(
 				"creator",
 				() -> {
 					if (objectDefinition.isUnmodifiableSystemObject()) {
-						return userId;
+						return null;
 					}
 
 					return MapUtil.getString(objectEntry, "userId");
```

Comment:
@devpedrotavares please inline if possible:

```
if (objectDefinition.isUnmodifiableSystemObject()) {
    return null;
}

return MapUtil.getString((Map<String, Object>)payloadJSONObject.get("objectEntry"), "userId");
```

Reply:
The `objectEntry` is used in another place as well